index.html

```diff
@@ -306,7 +306,7 @@ <h3 class="best">
           I'm always interested in hearing about new projects, so if you'd like
           to chat please get in touch.
         </p>
-        <form action="https://formspree.io/f/xnqwjqwy" method="post">
+        <form id="form">
```

Comment:
- [ ] I can see you made a lot of effort, unfortunately, you forget some but you are almost there,The contact form information should persistent to page reloads.but when we reloads the page all information disappeared, kindly fix this for your project to match all requirements :wink: